### DISCUSSION TITLE
Question custom composite component that is combination of two camel components

### DISCUSSION THREAD

**Discussion by jabhishek01** [apache/camel-karavan#1341] [Q&A] [answered]
### Ask a question

Question We are looking to create composite component that is combination of two camel components. Is this possible? Also can you please share us some examples ?

### Variant

None

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

None

**Comment by davsclaus** [comment]
What is your use-case and which 2 components do you want to combine.

**Reply by jabhishek01** [reply, marked as answer]
We were able to accomplish same using kamelets so now its not required. We were trying to create custom component for netty-htp and log but it requires karavan devmode image to be rebuilt with our jars so we avoided that and used kamelets instead.
